Header header
float32 Latitude
float32 Longitude
float32 Altitude
float32 HDOP
float32 UTM_easting
float32 UTM_northing
string UTC
int32 Zone
string Letter


